src/audio/igo_nr/igo_nr.c

```diff
@@ -585,38 +610,41 @@ static void igo_nr_print_config(struct comp_dev *dev)
 {
 	struct comp_data *cd = comp_get_drvdata(dev);
 
-	comp_dbg(dev, "  igo_params_ver	   %d",
+	comp_dbg(dev, "  igo_params_ver		%d",
```

Comment:
This formatting looks all unrelated to the channel change ? Should be a new patch.

Reply:
Yes this is just a format change. I will split it to new commit.